15:4: not a valid test operator:  
15:4: not a valid test operator: 12.4
21:4: not a valid test operator: (
21:4: not a valid test operator: 535.129.03
0it [00:00, ?it/s][rank0]: Traceback (most recent call last):
[rank0]:   File "/oscar/scratch/mxiangxi/CSCI1430-Final-Project-MedImage-Segmentation/segdiff/train_brats.py", line 182, in <module>
[rank0]:     main()
[rank0]:   File "/oscar/scratch/mxiangxi/CSCI1430-Final-Project-MedImage-Segmentation/segdiff/train_brats.py", line 104, in main
[rank0]:     TrainLoop(
[rank0]:   File "/oscar/scratch/mxiangxi/CSCI1430-Final-Project-MedImage-Segmentation/segdiff/improved_diffusion/train_util.py", line 206, in run_loop
[rank0]:     self.run_step(batch, cond)
[rank0]:   File "/oscar/scratch/mxiangxi/CSCI1430-Final-Project-MedImage-Segmentation/segdiff/improved_diffusion/train_util.py", line 259, in run_step
[rank0]:     self.forward_backward(batch, cond)
[rank0]:   File "/oscar/scratch/mxiangxi/CSCI1430-Final-Project-MedImage-Segmentation/segdiff/improved_diffusion/train_util.py", line 285, in forward_backward
[rank0]:     losses = compute_losses()
[rank0]:   File "/oscar/scratch/mxiangxi/CSCI1430-Final-Project-MedImage-Segmentation/segdiff/improved_diffusion/respace.py", line 97, in training_losses
[rank0]:     return super().training_losses(self._wrap_model(model), *args, **kwargs)
[rank0]:   File "/oscar/scratch/mxiangxi/CSCI1430-Final-Project-MedImage-Segmentation/segdiff/improved_diffusion/gaussian_diffusion.py", line 710, in training_losses
[rank0]:     model_output = model(x_t, self._scale_timesteps(t), **model_kwargs)
[rank0]:   File "/oscar/scratch/mxiangxi/CSCI1430-Final-Project-MedImage-Segmentation/segdiff/improved_diffusion/respace.py", line 124, in __call__
[rank0]:     return self.model(x, new_ts, **kwargs)
[rank0]:   File "/usr/local/lib/python3.10/dist-packages/torch/nn/modules/module.py", line 1511, in _wrapped_call_impl
[rank0]:     return self._call_impl(*args, **kwargs)
[rank0]:   File "/usr/local/lib/python3.10/dist-packages/torch/nn/modules/module.py", line 1520, in _call_impl
[rank0]:     return forward_call(*args, **kwargs)
[rank0]:   File "/usr/local/lib/python3.10/dist-packages/torch/nn/parallel/distributed.py", line 1523, in forward
[rank0]:     else self._run_ddp_forward(*inputs, **kwargs)
[rank0]:   File "/usr/local/lib/python3.10/dist-packages/torch/nn/parallel/distributed.py", line 1359, in _run_ddp_forward
[rank0]:     return self.module(*inputs, **kwargs)  # type: ignore[index]
[rank0]:   File "/usr/local/lib/python3.10/dist-packages/torch/nn/modules/module.py", line 1511, in _wrapped_call_impl
[rank0]:     return self._call_impl(*args, **kwargs)
[rank0]:   File "/usr/local/lib/python3.10/dist-packages/torch/nn/modules/module.py", line 1520, in _call_impl
[rank0]:     return forward_call(*args, **kwargs)
[rank0]:   File "/oscar/scratch/mxiangxi/CSCI1430-Final-Project-MedImage-Segmentation/segdiff/improved_diffusion/unet.py", line 488, in forward
[rank0]:     h = module(h, emb)
[rank0]:   File "/usr/local/lib/python3.10/dist-packages/torch/nn/modules/module.py", line 1511, in _wrapped_call_impl
[rank0]:     return self._call_impl(*args, **kwargs)
[rank0]:   File "/usr/local/lib/python3.10/dist-packages/torch/nn/modules/module.py", line 1520, in _call_impl
[rank0]:     return forward_call(*args, **kwargs)
[rank0]:   File "/oscar/scratch/mxiangxi/CSCI1430-Final-Project-MedImage-Segmentation/segdiff/improved_diffusion/unet.py", line 47, in forward
[rank0]:     x = layer(x)
[rank0]:   File "/usr/local/lib/python3.10/dist-packages/torch/nn/modules/module.py", line 1511, in _wrapped_call_impl
[rank0]:     return self._call_impl(*args, **kwargs)
[rank0]:   File "/usr/local/lib/python3.10/dist-packages/torch/nn/modules/module.py", line 1520, in _call_impl
[rank0]:     return forward_call(*args, **kwargs)
[rank0]:   File "/usr/local/lib/python3.10/dist-packages/torch/nn/modules/conv.py", line 460, in forward
[rank0]:     return self._conv_forward(input, self.weight, self.bias)
[rank0]:   File "/usr/local/lib/python3.10/dist-packages/torch/nn/modules/conv.py", line 456, in _conv_forward
[rank0]:     return F.conv2d(input, weight, bias, self.stride,
[rank0]: RuntimeError: Given groups=1, weight of size [128, 1, 3, 3], expected input[1, 4, 256, 256] to have 1 channels, but got 4 channels instead
0it [00:04, ?it/s]
srun: error: gpu2110: task 0: Exited with exit code 1
